lib/letter_avatar_app.rb

```diff
@@ -100,4 +105,11 @@ def self.call(env)
   def self.error(code, msg)
     [code, {'Content-Type' => 'text/plain'}, [msg]]
   end
+
+  private
+
+  def self.validate_cluster(query_string)
+    qry = Rack::Utils.parse_nested_query(query_string)
+    return qry['cluster'] == ENV['CLUSTER_NAME'] || nil
```

Comment:
```suggestion
   qry['cluster'] == ENV['CLUSTER_NAME']
```

We should be able to simplify the code to the above.

Reply:
If both are `nil`, this will result in `true` being returned, right?

Reply:
Oh that bug exists already. I'll fix it :)